daemon/config.go

```diff
@@ -80,7 +80,7 @@ type CommonConfig struct {
 	Hosts      []string         `json:"hosts,omitempty"`
 	LogLevel   string           `json:"log-level,omitempty"`
 	TLS        bool             `json:"tls,omitempty"`
-	TLSVerify  bool             `json:"tls-verify,omitempty"`
+	TLSVerify  bool             `json:"tlsverify,omitempty"`
 	TLSOptions CommonTLSOptions `json:"tls-opts,omitempty"`
```

Comment:
they are defined in the `CommonTLSOptions` struct. The conflict logic checks those fields when it finds inner structs like this one. The keys for the options are still called `tlscacert`, `tlscert` and `tlskey`.